src/app/app-routing.module.ts

```diff
@@ -2,13 +2,14 @@ import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
 import { HomeComponent } from './modules/home/components/home/home.component';
 import { CourseComponent } from './modules/course/components/course/course.component';
```

Comment:
Can you clean up the course component. Currently, the course template is inside course. Also if we are renaming to template can you also change the lesson component to match.

Reply:
I'll clean it up. I just need to delete the CourseComponent.